src/main/java/org/openhab/tools/analysis/report/ReportUtility.java

```diff
@@ -286,8 +286,7 @@ private String checkForErrors(String secondMergeResult, String reportLocation) {
                 result.append(log);
             }
         }
-        result.append("Detailed report can be found at: file").append(File.separator).append(File.separator)
-                .append(File.separator).append(reportLocation).append("\n");
+        result.append("Detailed report can be found at: file:///").append(reportLocation).append("\n");
```

Comment:
For some reason the "reportLocation" is a path relative to the root, i.e. it does not start with a slash. So my PR just replaces three `File.separator`s by three forward slashes and adds the missing colon. The result is a valid URL that you can copy from the log and paste into the browser.
If you want to go into a deeper refactoring, feel free to do so as a follow-up PR.